releasenotes/notes/qpy-v2-f1c380b40936cccf.yaml

```
@@ -0,0 +1,13 @@
+---
+features:
+  - |
+    QPY serialization is now capable of representing
+    :attr:`~qiskit.circuit.QuantumCircuit.global_phase` attributes of a
+    :class:`~qiskit.circuit.QuantumCircuit` object that are a ``int``,
+    :class:`~qiskit.circuit.Parameter` object, or
+    :class:`~qiskit.circuit.ParameterExpression` object. Previous versions of
+    QPY would only accept a :attr:`~qiskit.circuit.QuantumCircuit.global_phase`
+    that were ``float``s.
```

```suggestion
    that was a ``float``.
```

I _think_ Sphinx doesn't recognise ` `` ` followed by a word character, but not 100% sure.

---

Yeah, sphinx doesn't like that. Good catch.